src/sonic-yang-models/yang-models/sonic-extension.yang

```diff
@@ -0,0 +1,23 @@
+module sonic-extension {
```

Comment:
Please remove this file. It is mistakenly added here and it is part of yang-templates